tests/test-basic.sh

```diff
@@ -21,6 +21,13 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
+# This test looks for specific localized strings. Prefer a UTF-8 locale
+if locale -a | grep '^C\.UTF-8$' >/dev/null; then
```

Comment:
> Does Debian support systems without C.UTF-8 ?

Not any more (squeeze was the last suite without it, and recently left LTS support), but I can't speak for other distros. The plain C locale is POSIX-specified and should always exist, but occasionally causes issues with non-ASCII strings (particularly when Python is involved), which is why Debian now generates C.UTF-8.